sdk/schemaregistry/schema-registry/src/schemaRegistryClient.ts

```diff
@@ -45,20 +46,11 @@ export class SchemaRegistryClient {
     this.client = new GeneratedSchemaRegistryClient(endpoint, { ...pipeline, endpoint });
   }
 
-  /**
-   * Registers a schema.
-   *
-   * If schema of specified name does not exist in the specified group, a schema
-   * is created at version 1. If schema of specified name exists already in
-   * specified group, schema is created at latest version + 1.
-   *
-   * @param schema Schema to register.
-   * @return Registered schema's ID.
-   */
+  /** @inheritdoc */
```

Comment:
I don't think our doc pipeline is smart enough to understand `@inheritdoc`, sorry to say

Reply:
OK, copy/paste it is.